src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/Cells/DiaryDayAddTableViewCell.swift

```diff
@@ -0,0 +1,26 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class DiaryDayAddTableViewCell: UITableViewCell {
+
+	// MARK: - Internal
+
+	func configure(entryType: DiaryEntryType) {
+		switch entryType {
+		case .contactPerson:
+			label.text = AppStrings.ContactDiary.Day.addContactPerson
+		case .location:
+			label.text = AppStrings.ContactDiary.Day.addLocation
+		}
+
+		accessibilityTraits = [.button]
+	}
```

Comment:
Basically the to follow MVVM the cells should have a ViewModel (aka CellViewModel) too.

Reply:
Add a cell model here 😊